core/controllers/learner_dashboard_test.py

```diff
@@ -238,6 +246,31 @@ def test_get_learner_dashboard_ids(self):
             learner_dashboard_activity_ids['collection_playlist_ids'],
             [self.COL_ID_3])
 
+    def test_get_with_threads_update_thread_summaries(self):
```

Comment:
This test name is pretty confusing, perhaps `test_get_threads_after_updating_thread_summaries` instead?

Reply:
Done